tests/ci/integration/run_python_integration.sh

```diff
@@ -91,8 +176,6 @@ function python_run_tests() {
 #     authentication portion of that protocol.
 #   - Modify the ssl module's backing C code to account for AWS-LC's divergent
 #     function signature and return value for |sk_SSL_CIPHER_find|
-#
-# TODO: Remove these patches when we make an upstream contribution.
```

Comment:
Nit: Are all of the call outs above still applicable? I was under the impression that we finished tweaking some of the places in AWS-LC for better adoption.

Reply:
They're still applicable for older versions of CPython, namely 3.10. Unfortunately 3.10 is now [only taking security patches](https://github.com/python/cpython/pull/117229#issuecomment-2041163376).

Reply:
ahh understood